### PR TITLE
New version for solving authentication problem

### PR DESCRIPTION
**Pull request by rshunim**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-13510)

## Description
New docker image, and a small change - 'autologin' was replaced to 'basic' when user use username and password
https://github.com/splunk/splunk-sdk-python/issues/219#issuecomment-611478090

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
1. please add release notes and bump version.
2. add the same update to splunkPy-pre release integration.

**Comment by ilappe** [comment]
@rshunim 
can you please add here link to Splunk SDK documentation that mention the "basic" attribute in the splunklib.client?

**Comment by ilappe** [comment]
discussion about the `basic=true `
https://github.com/splunk/splunk-sdk-python/issues/219#issuecomment-611478090

**Review by ilappe — APPROVED** [review]

**Comment by rshunim** [comment]
> @rshunim can you please add here link to Splunk SDK documentation that mention the "basic" attribute in the splunklib.client?
https://github.com/splunk/splunk-sdk-python/issues/219#issuecomment-611478090

File: Packs/SplunkPy/ReleaseNotes/2_4_5.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### SplunkPy
+- Updated the Docker image to: *demisto/splunksdk:1.0.0.32388*.
```
Comment (yuvalbenshalom) [inline Packs/SplunkPy/ReleaseNotes/2_4_5.md:4]:
```suggestion
- Fixed an issue where the integration authentication failed intermediately.
- Updated the Docker image to: *demisto/splunksdk:1.0.0.32388*.
```

File: Packs/SplunkPyPreRelease/ReleaseNotes/1_0_12.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### SplunkPy Prerelease (Beta)
+- Updated the Docker image to: *demisto/splunksdk:1.0.0.32388*.
```
Comment (yuvalbenshalom) [inline Packs/SplunkPyPreRelease/ReleaseNotes/1_0_12.md:4]:
```suggestion
- Fixed an issue where the integration authentication failed intermediately.
- Updated the Docker image to: *demisto/splunksdk:1.0.0.32388*.
```

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
added a line to state the fix in the release notes.

**Review by yuvalbenshalom — APPROVED** [review]

**Pull request merged**